docs/user-guide/kubectl-overview.md

```diff
@@ -263,7 +269,10 @@ $ kubectl delete -f pod.yaml
 // Delete all the pods and services that have the label name=<label-name>.
 $ kubectl delete pods,services -l name=<label-name>
 
-// Delete all pods.
+// Delete all the pods and services that have the label name=<label-name>, including uninitialized ones.
+$ kubectl delete pods,services -l name=<label-name> --include-uninitialized
+
+// Delete all pods, including uninitialized ones.
```

Comment:
Not sure if --all deletes uninitialized? Commenting just to doublecheck.

Reply:
Yeah, I think so.

Reply:
> Not sure if --all deletes uninitialized?

@ahmetb @caesarxuchao If flag `--all` is set, uninitialized ones are included as default.

Also kubernetes/kubernetes#51186 solved the bug when deleting uninitialized resources.

I've re-tested using master branch, it works as expected.